torch/testing/_internal/distributed/ddp_under_dist_autograd_test.py

```diff
@@ -287,11 +282,7 @@ def set_shutdown_signal():
     with shutdown_signal:
         shutdown_signal.notify()
 
-@unittest.skipIf(
-    TEST_WITH_ASAN,
-    "Skip ASAN as torch + multiprocessing spawn have known issues",
-)
-class TestDdpUnderDistAutograd(MultiProcessTestCase, RpcAgentTestFixture):
+class TestDdpUnderDistAutograd(RpcAgentTestFixture):
```

Comment:
IIUC, since after this change, tests should be directly launched from this class, shall we rename it (e.g., to `DdpUnderDistAutogradTest`) to make it clear?

Reply:
I'm up for renaming it, as it makes it consistent with the convention in other classes.

> tests should be directly launched from this class

Not sure I understand. After this whole stack (but not yet as of this commit) the classes defined in this file (and in the other files under `torch/testing/_internal/distributed`, which includes `RpcTest`, `DistAutogradTest`, ...) will be just "mix-ins": they will define the test methods, but in order to actually be able to execute them they must be mixed together with the MultiProcessTestCase class and with a subclass of the RpcAgentTestFixture (that class will be abstract, and each subclass will specialize it for a specific agent). This final "mixing" of the three classes will happen, for example, in `test/distributed/rpc/test_process_group_agent.py`.